assets/pi.yaml

```diff
@@ -0,0 +1,23 @@
+---
+
+- type: application
+  name: PI
```

Comment:
I think it is `Pi` but check

Reply:
Yup, Pi should be the right capitalization!